sc/src/test/java/uk/ac/ebi/atlas/metadata/CellMetadataDaoIT.java

```diff
@@ -147,6 +150,24 @@ void invalidCellIdAndExperimentAccessionHasNoFactorFields() {
         assertThat(subject.getFactorFieldNames(experimentAccession, cellId)).isEmpty();
     }
 
+    @Test
+    void experimentWithMissingValuesReturnsNotAvailable() {
+        String experimentAccession = "E-GEOD-71585";
+
+        ImmutableList<String> cellIdsWithMissingValues = ImmutableList.of(
+                "SRR2138737",
+                "SRR2140225",
+                "SRR2139550",
+                "SRR2139566");
+
+        Map<String, String> result = subject.getMetadataValueForCellIds(experimentAccession,
+                SingleCellAnalyticsCollectionProxy.CHARACTERISTIC_INFERRED_CELL_TYPE,
+                cellIdsWithMissingValues);
+
+        assertThat(result.values()).containsOnly("not available");
```

Comment:
Rather than hard-coding the string in the class and the test, I think it’s better to have the tested class have a `static final String` variable that can be checked by the test class.